build/IPDK_Container/Dockerfile

```diff
@@ -67,12 +67,12 @@ WORKDIR /root
 COPY ./scripts scripts
 
 FROM base AS p4-ovs-container
+ARG KEEP_SOURCE_CODE
```

Comment:
Can you make similar changes to the Ubuntu20.04/Dockerfile and Makefile as well? I merged some changes today in the Dockerfile, so make sure to pick those up first.

Reply:
@mestery, Planning to pick up the Ubuntu container immediately once after the review process and considerable testing is done for Fedora container.

Reply:
Done, made changes on top of #31 for Ubuntu and validated with both KEEP_SOURCE_CODE=YES  and KEEP_SOURCE_CODE=NO options.